### ISSUE TITLE
Add filter to transform postgresql to postgres

### ISSUE THREAD

**Issue by cry** [secedu/flask-core#13]
Logs:

```
web_service_1  | [2019-03-25 05:07:02,564] ERROR in IsolationMiddleware: Couldn't import database isolation adapter postgresql.Postgresql
web_service_1  | [2019-03-25 05:07:02,564] ERROR in IsolationMiddleware: Couldn't import database isolation adapter postgresql.Postgresql
web_service_1  | Auth disabled, auto disabling database isolation
web_service_1  | [2019-03-25 05:07:02,665] ERROR in IsolationMiddleware: Couldn't import database isolation adapter postgresql.Postgresql
web_service_1  | Auth disabled, auto disabling database isolation
web_service_1  | [2019-03-25 05:07:02,694] ERROR in IsolationMiddleware: Couldn't import database isolation adapter postgresql.Postgresql
```

**Comment by cry** [comment]
@zainafzal08 #goodfirstissue

**Comment by cry** [comment]
Closed by #14
